mne/viz/evoked.py

```diff
@@ -60,6 +61,40 @@ def _butterfly_on_button_press(event, params):
     params['need_draw'] = False
 
 
+def _butterfly_onselect(xmin, xmax, ch_types, evoked):
+    """Function for drawing topomaps from the selected area."""
+    import matplotlib.pyplot as plt
+    times = evoked.times
+    xmin *= 0.001
+    xmin = times.flat[np.abs(times - xmin).argmin()]
+    minidx = np.where(times == xmin)[0][0]
```

Comment:
Unless I'm missing something, instead of these two lines, the following seems a bit clearer and involves less computation:
```
minidx = np.abs(times-xmin).argmin()
xmin = times[minidx]
```
Same for the one below.